### ISSUE TITLE
Fix Date Field

### ISSUE THREAD

**Issue by brendan-s1** [SummitRoute/csp_security_mistakes#9]
Should you have a Discover Date and a Fix Date field? Or maybe i should be asking - what happens to these issues once they are addressed? Do they go off this page? If not, i suggest a fix date be added.

**Comment by 0xdabbad00** [comment]
Issues remain on this page whether they are resolved or not.  It makes sense to add whether the issue has been resolved or not. Identifying when it has been resolved may be difficult though, and generally I don't think people care too much (at least not for the work involved, although if someone wants to prove me wrong by submitting a PR with the fixed dates, I'm happy to accept it. :) )

**Comment by brendan-s1** [comment]
Yeah i think at least a 'status' field of fixed or not fixed would be helpful.
